app/src/main/java/com/akshay/newsapp/NewsApp.kt

```diff
@@ -1,22 +1,15 @@
 package com.akshay.newsapp
 
 import android.app.Application
-import com.akshay.newsapp.di.base.AppInjector
-import dagger.android.AndroidInjector
-import dagger.android.DispatchingAndroidInjector
-import dagger.android.HasAndroidInjector
-import javax.inject.Inject
+import com.akshay.newsapp.koin.AppInjector
 
-class NewsApp : Application(), HasAndroidInjector {
+class NewsApp : Application() {
 
-    @Inject
-    lateinit var androidInjector: DispatchingAndroidInjector<Any>
+	override fun onCreate() {
+		super.onCreate()
 
-    override fun onCreate() {
-        super.onCreate()
-        // Init DI magic ✨
-        AppInjector.init(this)
-    }
+		// Init DI magic ✨
+		AppInjector.init(this@NewsApp)
```

Comment:
Maybe you can make it more concise and remove the extra line at the bottom
```suggestion
		AppInjector.init(this)
```